### PR TITLE
Fix multimodule IC with JS KLIB compiler

### PR DESCRIPTION
**Pull request by altsvetkov**

### CODE REVIEW

File: compiler/util-klib-metadata/src/org/jetbrains/kotlin/library/metadata/KlibMetadataModuleDescriptorFactory.kt [hunk 1]
```diff
@@ -31,22 +32,26 @@ interface KlibMetadataModuleDescriptorFactory {
...
+    )
 
     fun createDescriptorOptionalBuiltIns(
         library: KotlinLibrary,
         languageVersionSettings: LanguageVersionSettings,
         storageManager: StorageManager,
         builtIns: KotlinBuiltIns?,
-        packageAccessHandler: PackageAccessHandler?
+        packageAccessHandler: PackageAccessHandler?,
+        lookupTracker: LookupTracker
```
[the remark this answers is not in the document]
Comment (altsvetkov) [reply compiler/util-klib-metadata/src/org/jetbrains/kotlin/library/metadata/KlibMetadataModuleDescriptorFactory.kt]: @anton-bannykh yes, that was my intent. I don't see any benefits for the default value in this particular case. The `DO_NOTHING` tracker is no more default that `LookupTrackerImpl`, there is no way to tell which one should be used without taking context into account.

[the remark this answers is not in the document]
Comment (anton-bannykh) [reply compiler/util-klib-metadata/src/org/jetbrains/kotlin/library/metadata/KlibMetadataModuleDescriptorFactory.kt]: Ok, got it

**Comment by anton-bannykh** [comment]
LGTM

**Comment by altsvetkov** [comment]
Merged manually

**Pull request closed without merging**